WikiDrips/WikiDoc.swift

```diff
@@ -8,10 +8,10 @@
 
 import Foundation
 
-public class WikiDoc
+public struct WikiDoc
 {
     public let title: String
-    public let pubDate: Date
+    public let pubDate: String?
```

Comment:
I think it's better if your app models are strongly typed. In this case `Date?` instead of `String?`